packages/account/src/providers/utils/sleep.ts

```diff
@@ -6,3 +6,16 @@ export function sleep(time: number) {
     }, time);
   });
 }
+
+export async function sleepUntilTrue(
```

Comment:
Is this only a test utility? May be better placed in a `test` directory.

Reply:
Moved to `@fuel-ts/utils/test-utils`: https://github.com/FuelLabs/fuels-ts/pull/1356/commits/127f7b221e65590600209e96769d3c85ce95a2e5